Core/src/org/sleuthkit/autopsy/mainui/nodes/TreeChildFactory.java

```diff
@@ -198,6 +223,15 @@ protected void addNotify() {
         registerListeners();
         super.addNotify();
     }
+    
+    public static <T> TreeItemDTO<T> getUpdatedTreeData(TreeItemDTO<T> original, T updatedData) {
```

Comment:
Can this be package scope?  Shoudl probably be named something createTreeItemDTO().

Reply:
And needs a comment.